src/interface/substrate_query.js

```diff
@@ -428,6 +448,13 @@ module.exports = {
                 } catch (e) {
                     return {'error': e.toString()};
                 }
+	    case 'staking/payee':
```

Comment:
In this case since `param2` is a requirement we must check whether the user entered it or not. I suggest the following implementation:

```
	    case 'staking/payee':
                try {
                    if (!param2) {
                        return {'error': 'You did not enter the stash '
                                + 'address that needs to be queried'};
                    }
                    return {'result': await getStakingPayee(api,
                            param2)};
                } catch (e) {
                    return {'error': e.toString()};
                }
```

Reply:
Would appreciate if you could test the above code out.